lib/response.js

```diff
@@ -22,7 +22,7 @@ module.exports = (function () {
             this.handler.response = buildSpeechletResponse({
                 sessionAttributes: this.attributes,
                 output: getSSMLResponse(speechOutput),
-                reprompt: getSSMLResponse(repromptSpeech),
+                reprompt: getSSMLResponse(repromptSpeech || speechOutput),
```

Comment:
I understand and a good voice user experience should include a reprompt when prompting a user, but I worry that this change to automatically add reprompt would not be obvious to a developer.